timelock-server/src/integTest/java/com/palantir/atlasdb/timelock/LockWatchEventIntegrationTest.java

```diff
@@ -359,6 +387,30 @@ private Set<LockDescriptor> getDescriptors(TableReference tableRef, Cell... cell
                 .collect(Collectors.toSet());
     }
 
+    private List<Optional<LockRequestMetadata>> lockAndGetAllLockEventMetadataForNewTransaction(
```

Comment:
Maybe worth just returning the `Optional<LockRequestMetadata>`, seeing as we only expect a single event anyways?

Reply:
`lockAndGetEventMetadataFromLockWatchLog` ?